Clarify which contributing docs refer to what

ecraig12345 opened the pull request:
`.github/CONTRIBUTING.md` and a few related files under `.github` were copied over from the old fluent repo without modification, despite not being applicable to large parts of the converged repo (and now having a lot of incorrect paths/links).

This PR attempts to clarify that those files only refer to...the things under `packages/fluentui`, however we want to say that. Initially I'm using the disambiguation language "______ for `@fluentui/react-northstar` (and others under `packages/fluentui`)" in the headings, but am **very** open to suggestions.

(While we should obviously do a full merge of contributing docs at some point, I'm NOT going to take that on right now.)

A much simpler change is that I updated all the paths/links the documents reference, and fixed a handful of stray old package names or URLs which snuck through in other places.

### UPDATE

I moved all those files under `packages/fluentui`. I also added a readme specific to that folder, copied from current and previous versions of the microsoft/fluent-ui-react readme.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12384)

dzearing commented on .github/CONTRIBUTING.md:
```diff
@@ -90,13 +96,13 @@ For example, to make an icon-only Button accessible, `aria-label` prop needs to
 - using the screen reader with or without virtual cursor
 
-Fluent UI uses Office UI Fabric [FocusZone][8] for basic TAB and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).
+Fluent UI v0 uses the [FocusZone][8] component for basic tab and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).
```
```suggestion
Fluent UI uses the [FocusZone][8] component for basic tab and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).
```

dzearing commented on .github/CONTRIBUTING.md:
```diff
@@ -12,59 +14,63 @@
 ## Accessibility
 
-Fluent UI implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-\* properties) as well as handle keyboard interaction and focus. Every accessible component has a default behavior, which can be overridden using the `accessibility` prop. You can choose a behavior from the ones provided by Fluent UI or you can implement a new b ...
+Fluent UI v0 implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-\* properties) as well as handle keyboard interaction and focus. Every accessible component has a default behavior, which can be overridden using the `accessibility` prop. You can choose a behavior from the ones provided by Fluent UI or you can implement a ne ...
```
```suggestion
`@fluentui/react-northstar` implements accessibility using accessibility behaviors. The behaviors add attributes to the DOM elements (mainly role and aria-\* properties) as well as handle keyboard interaction and focus. Every accessible component has a default behavior, which can be overridden using the `accessibility` prop. You can choose a behavior from the ones provided by Fluent UI or you can implement a new behavior.
```

dzearing commented on .github/CONTRIBUTING.md:
```diff
@@ -90,13 +96,13 @@ For example, to make an icon-only Button accessible, `aria-label` prop needs to
+Fluent UI v0 uses the [FocusZone][8] component for basic tab and arrow key focus handling. To use the focus zone, you can use the `focusZone` configuration in the behavior (for example see [MenuItemBehavior][9]).
 
-Focused component needs to be clearly visible. This is handled in Fluent UI by focus indicator functionality. Focus indicator will be displayed only if the application is in keyboard mode. Application switches to keyboard mode when a key relevant to navigation is pressed. It disables keyboard mode on mouse click events.
+Focused component needs to be clearly visible. This is handled in Fluent UI v0 by focus indicator functionality. Focus indicator will be displayed only if the application is in keyboard mode. Application switches to keyboard mode when a key relevant to navigation is pressed. It disables keyboard mode on mouse click events.
```
```suggestion
Focused component needs to be clearly visible. This is handled in `@fluentui/react-northstar` by focus indicator functionality. Focus indicator will be displayed only if the application is in keyboard mode. Application switches to keyboard mode when a key relevant to navigation is pressed. It disables keyboard mode on mouse click events.
```

dzearing commented on .github/CONTRIBUTING.md:
````diff
@@ -12,59 +14,63 @@
+
 ```sh
-yarn start                 // run doc site
+yarn start                 # start doc site: choose `@fluentui/docs` for v0
````
```suggestion
yarn start                 # start doc site: choose `@fluentui/docs` for `@fluentui/react-northstar`
```

layershifter commented on .github/CONTRIBUTING.md:
```diff
@@ -1,4 +1,6 @@
-# CONTRIBUTING
+# Contributing to `@fluentui/react-northstar` (and others under `packages/fluentui`)
```
Should we move this file to `packages/fluentui`, too?

layershifter commented on .github/CONTRIBUTING.md:
```diff
@@ -110,17 +116,22 @@ We are using [Lerna][14] to manage our packages and [Yarn Workspaces][15] to lin
 #### Run `lerna create`
 
 You should to run `lerna create` command to create a new package
+-->
```
```suggestion
```

layershifter commented on .github/CONTRIBUTING.md:
````diff
@@ -196,6 +207,7 @@ module.exports = {
   name: '__DIRECTORY_NAME__',
 };
 ```
+-->
````
Let's also remove this section (currently commented)

layershifter commented:
I suggest to move these .md files to `packages/fluentui`. I think that it will be the best option to add clarity 😺

ecraig12345 commented:
> I suggest to move these .md files to `packages/fluentui`. I think that it will be the best option to add clarity 😺

Oh yeah...I suppose that would be a nice logical solution. 😆

ecraig12345 commented:
Per @layershifter's suggestion I moved all those files under `packages/fluentui`. I also added a readme specific to that folder, copied from current and previous versions of the microsoft/fluent-ui-react readme.
